### PR TITLE
Remove limited avail notes for security groups and make a few other s…

### PR DESCRIPTION
**Pull request by catlook**
…ec grp updates

closes issue #2189

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-c557f1756d/how-to/). :bow:

File: content/cloud-networks/cloud-networks-faq.md [hunk 1]
```diff
@@ -150,7 +142,7 @@ Traffic that matches the new security group rule is allowed to go through.
 
 #### Can traffic be blocked or denied based on a security group rule?
 
-No. Traffic that matches a rule is permitted, and any traffic that is not part of the ruleset for that security group is denied or blocked. Because of OpenStack API design requirements, you cannot specify that traffic matching a rule should be denied. The security groups API is a whitelist. Thus, traffic that doesn't match any of the rules in the whitelist is automatically blacklisted.
+Traffic that matches a rule is permitted. Any traffic that is not part of the ruleset for that Security Group is denied / blocked. There is no way to specify that traffic matching a rule should be denied. This is how the OpenStack Security Groups API was designed. Hence the Security Groups API is a white-list. Traffic that doesn't match any of the rules in the white-list is automatically black-listed.
```
Comment (DonNateR) [inline content/cloud-networks/cloud-networks-faq.md]: Small nit for the third and fourth sentences of the paragraphs. I would combine the two sentences. For example "OpenStack Security Groups API is designed as a white-list."

Comment (DonNateR) [inline content/cloud-networks/cloud-networks-faq.md]: "....Security Group is denied or blocked."

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-5f7b02c25d/how-to/)

**Comment by catlook** [comment]
It's ready top go but delayed until aug 28 at earliest.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8c3a931c24/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5e8b7fcba6/how-to/)

**Pull request merged**